sourcetransaction/client.go

```diff
@@ -0,0 +1,63 @@
+// Package sourcetransaction provides the /source/transactions APIs.
+package sourcetransaction
+
+import (
+	"fmt"
+
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+)
+
+// Client is used to invoke /sources/:source_id/transactions APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// List returns a list of source transactions.
+// For more details see https://stripe.com/docs/api#retrieve_source.
+func List(params *stripe.SourceTransactionListParams) *Iter {
+	return getC().List(params)
+}
+
+func (c Client) List(params *stripe.SourceTransactionListParams) *Iter {
+	var body *form.Values
+	var lp *stripe.ListParams
+	var p *stripe.Params
+
+	if params != nil {
+		body = &form.Values{}
+		form.AppendTo(body, params)
+		lp = &params.ListParams
+		p = params.ToParams()
+	}
+
+	return &Iter{stripe.GetIter(lp, body, func(b *form.Values) ([]interface{}, stripe.ListMeta, error) {
+		list := &stripe.SourceTransactionList{}
```

Comment:
It's probably not convention elsewhere, but it might be worth failing fast here by returning an error if `params.Source` wasn't set. Currently I believe it'll fail with kind of a confusing `couldn't find URL /sources//source_transactions` 404 (or something of the sort).